app/controllers/api/account.php

```diff
@@ -3593,19 +3593,17 @@
 
         $otp = (match ($type) {
             Type::TOTP => new TOTP(),
-            default => throw new Exception(Exception::GENERAL_UNKNOWN, 'Unknown type.')
+            default => throw new Exception(Exception::GENERAL_ARGUMENT_INVALID, 'Unknown type.') // Ideally never happens if param validator stays always in sync
         });
 
         $otp->setLabel($user->getAttribute('email'));
         $otp->setIssuer($project->getAttribute('name'));
 
-        $backups = Type::generateBackupCodes();
-
         $authenticator = TOTP::getAuthenticatorFromUser($user);
 
         if ($authenticator) {
             if ($authenticator->getAttribute('verified')) {
-                throw new Exception(Exception::GENERAL_UNKNOWN, 'Authenticator already exists on this account.');
+                throw new Exception(Exception::GENERAL_UNKNOWN, 'Authenticator already exists on this account.'); // TODO: Type this exception
```

Comment:
Reminder

Reply:
Thanks, missed that one.

Reply:
All good, double-checked all list endpoints in user& account contorller

Reply:
Fixed